src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -292,9 +292,7 @@ In Chrome, the image must respond to certain criteria:
 
 {% Aside 'gotchas' %}
 The `description` and `screenshots` properties are currently used only in Chrome
-for Android when a user wants to install your app. The experimental flag
-`about://flags/#mobile-pwa-install-use-bottom-sheet` flag must be enabled in
-Chrome 90.
+for Android when a user wants to install your app.
```

Comment:
Have any other vendors made a comment on this—is it worth linking here?

Reply:
None sadly yet.